README.md

```diff
@@ -45,7 +47,7 @@ Every 2 weeks on Fridays at 4pm CET
 
 ## Where
 
-- on-line on [zoom](https://zoom.us/)
+- on-line on [zoom](https://zoom.us/), ID of the meeting is `904-769-991`.
```

Comment:
I created a recurring Zoom meeting (the ID used was not durable). New ID is `974-346-848`.